language/go/testdata/tests_per_file/BUILD.want

```diff
@@ -0,0 +1,19 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_test")
```

Comment:
Could you make this test more realistic by including a library that would need to be added as a dep to each test?

Reply:
Sure, but deps aren't resolved by `TestGenerateRules` so they'll just show up in `_gazelle_imports`, not `deps`.